attestation-container/protobuf/attestation-container.proto

```diff
@@ -18,4 +18,5 @@ message FetchAttestationRequest {
 message FetchAttestationReply {
   bytes attestation = 1;
   bytes attestation_endorsement_certificates = 2;
+  string uvm_endorsement = 3;
```

Comment:
Should this be `bytes` here too, for consistency with the other members? This means we would have to decode the base64-encoded UVM endorsements before returning them.

Reply:
Makes sense, added code to decode base64 string and its test